articles/quickstart/spa/vuejs/_includes/_centralized_login.md

```diff
@@ -2,475 +2,441 @@
 
 <%= include('../../_includes/_login_preamble', { library: 'Vue.js' }) %>
 
-### Create an Authentication Service
+## Create a Sample Application
 
-The best way to manage and coordinate the tasks necessary for user authentication is to create a reusable service. With the service in place, you will be able to call its methods throughout your application. The name for it is at your discretion, but in these examples it will be called `AuthService` and the filename will be `authService.js`. An instance of the `WebAuth` object from **auth0.js** can be created in the service.
-
-Create a service and instantiate `auth0.WebAuth`. Provide a method called `login` which calls the `authorize` method from auth0.js.
+::: note
+The following tutorial creates a new Vue application using the [Vue CLI](https://cli.vuejs.org/guide/), and presents some common ways to build Vue applications, in terms of its structure and naming conventions. If you are using this guide to integrate the Auth0 SDK into your Vue application, you may need to adjust some of the steps to suit your scenario.
+:::
 
-```js
-// src/auth/authService.js
-
-import auth0 from 'auth0-js';
-import EventEmitter from 'events';
-import authConfig from '../../auth_config.json';
-
-const webAuth = new auth0.WebAuth({
-  domain: authConfig.domain,
-  redirectUri: `<%= "${window.location.origin}" %>/callback`,
-  clientID: authConfig.clientId,
-  responseType: 'id_token',
-  scope: 'openid profile email'
-});
+If you don't already have an existing application, you can create one using the [Vue CLI](https://cli.vuejs.org/guide/) tool. Using the terminal, find a location on your drive where you want to create the project and run the following commands:
 
-class AuthService extends EventEmitter {
+```bash
+# Install the CLI
+npm install -g @vue/cli
 
-  // Starts the user login flow
-  login(customState) {
-    webAuth.authorize({
-      appState: customState
-    });
-  }
+# Create the application using the Vue CLI.
+# When asked to pick a preset, accept the defaults
```

Comment:
I think you're getting yarn as an option because you've got it installed - the article will work fine with either.

One thing that has come out of the quickstart principals talks (from the few I've been involved in) is that we should default to `npm` when talking about it on the docs and assume that's what the reader will be using. If an option for `yarn` is presented, then it's likely that the reader will probably just use that if they so chose, and would be comfortable translating the commands as they come up.